asteroid/data/musdb18_dataset.py

```diff
@@ -213,3 +213,26 @@ def get_tracks(self):
                         'path': track_path,
                         'min_duration': None
                     })
+
+    def get_infos(self):
+        """ Get dataset infos (for publishing models).
+
+        Returns:
+            dict, dataset infos with keys `dataset`, `task` and `licences`.
+        """
+        infos = dict()
+        infos['dataset'] = self.dataset_name
+        infos['task'] = 'enhancement'
+        infos['licenses'] = [musdb_license]
+        return infos
+
+
+musdb_license = dict(
+    title='MUSDB18',
+    title_link='https://sigsep.github.io/datasets/musdb.html',
+    author='SigSep',
+    author_link='',
+    license='CC BY-NC-SA 4.0',
+    license_link='https://creativecommons.org/licenses/by-nc-sa/4.0/',
+    non_commercial=True,
+)
```

Comment:
@faroit & @aliutkus could you please review that these informations are correct?

Reply:
```suggestion
musdb_license = dict(
)
```
For now, I'll keep it like this and we'll get to it when we need to share asteroid models from MUSDB